README.md

```diff
@@ -1,5 +1,7 @@
 <div align="center">
-<h1>fozzie</h1>
+<h1>Fozzie</h1>
+
+[This repo has been moved to fozzie-components](https://github.com/justeat/fozzie-components/tree/fozzie-migrate/packages/tools/fozzie)
```

Comment:
Looks like this URL goes to a 404 page atm

Reply:
Looks like the branch was `migrate-fozzie` not `fozzie-migrate`, plus it will have been deleted after the merge, so we can set this to master now.

Reply:
Yes please 😄  Would be good to pick this up Monday @siggerzz, so we can deprecate this repo